app/services/foreman_discovery/node_api/power_legacy_direct_service.rb

```diff
@@ -1,15 +0,0 @@
-module ForemanDiscovery::NodeAPI
-  # legacy /reboot call direct
-  class PowerLegacyDirectService < NodeResource
-    def initialize(args)
-      super args
-      raise ArgumentError, "Legacy direct service only supports http scheme" if scheme != 'http'
-      raise ArgumentError, "Legacy direct service only supports port 8443" if port != 8443
-      Foreman::Deprecation.deprecation_warning("1.11", "Discovery legacy API will be removed, use the new Power API")
-    end
-
-    def reboot
-      ::ProxyAPI::BMC.new(:url => url).power :action => "cycle"
-    end
-  end
-end
```

Comment:
Also remove the legacy_proxied.service.rb file please.

Reply:
@lzap  good catch, thanks :)